composer.lock

```diff
@@ -470,24 +470,24 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/PHPCompatibility/PHPCompatibilityParagonie.git",
-                "reference": "b862bc32f7e860d0b164b199bd995e690b4b191c"
```

Comment:
The changes to this file seem unrelated. Could you remove them?

Reply:
I think I goofed this up, and then when trying to restore, accidentally also updated the composer.lock in the jetpack plugin's directory. What's the best way to fix this one before I dig a deeper hole?

Reply:
The easiest way to reset the files is to check out the versions of the files from the merge-base of your branch and master
```
git checkout $(git merge-base HEAD master) composer.lock projects/plugins/jetpack/composer.lock
```

Reply:
Beautiful thank you, should be fixed now!